node/clients/checkout.ts

```diff
@@ -0,0 +1,42 @@
+import type { InstanceOptions, IOContext, RequestConfig } from '@vtex/api'
+import { JanusClient } from '@vtex/api'
+
+export default class Checkout extends JanusClient {
+  constructor(ctx: IOContext, options?: InstanceOptions) {
+    super(ctx, {
+      ...options,
+      headers: {
+        ...(options?.headers ?? {}),
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+        'X-Vtex-Use-Https': 'true',
+      },
+    })
+  }
+
+  public getNearestPickupPoints = (lat: string, long: string) => {
+    return this.get<CheckoutOutput>(
+      this.routes.getNearestPickupPoints(lat, long),
+      {
+        metric: 'checkout-nearestPickupPoints',
+      }
+    )
+  }
+
+  protected get = async <T>(url: string, config: RequestConfig = {}) => {
+    try {
+      return await this.http.get<T>(url, config)
+    } catch (e) {
+      return e
+    }
+  }
```

Comment:
This is gonna prevent us to throw errors when they happen. IMO, not a good approach. We can call `this.http.get` inside the methods.